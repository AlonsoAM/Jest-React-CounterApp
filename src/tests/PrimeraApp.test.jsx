import React from "react";
const { render } = require("@testing-library/react");
const { default: PrimeraApp } = require("../PrimeraApp");

describe("Prueba en Primera App", () => {
    test("Debe mostrar el mensaje: Hola soy Goku", () => {
        const saludo = "Hola soy Goku";
        const { getByText } = render(<PrimeraApp saludo={saludo} />);

        expect(getByText(saludo)).toBeInTheDocument();
    });
});
