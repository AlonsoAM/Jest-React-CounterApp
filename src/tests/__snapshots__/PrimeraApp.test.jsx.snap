// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en Primera App Debe de mostrar PrimerApp correctamente 1`] = `
<Fragment>
  <h1>
    Hola soy Goku
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
